packages/matter.js/src/cluster/ActionsCluster.ts

```diff
@@ -0,0 +1,345 @@
+ * 
+ * @see {@link MatterCoreSpecificationV1_0} § 9.14.7
+ */
+const CommandBits = TlvBitmap(TlvUInt16, {
+    /** This command triggers an action (state change) on the involved endpoints. */
+    instantAction: BitFlag(0),
+
+    /** This command triggers an action (state change) on the involved endpoints, with a specified time to transition from the current state to the new state. */
+    instantActionWithTransition: BitFlag(1),
+
+    /** This command triggers the commencement of an action on the involved endpoints. */
+    startAction: BitFlag(2),
+
+    /** This command triggers the commencement of an action (with a duration) on the involved endpoints. */
+    startActionWithDuration: BitFlag(3),
+
+    /** This command stops the ongoing action on the involved endpoints. */
+    stopAction: BitFlag(4),
+
+    /** This command pauses an ongoing action. */
+    pauseAction: BitFlag(5),
+
+    /** This command pauses an ongoing action with a duration. */
+    pauseActionWithDuration: BitFlag(6),
+
+    /** This command resumes a previously paused action. */
+    resumeAction: BitFlag(7),
+
+    /** This command enables a certain action or automation. */
+    enableAction: BitFlag(8),
+
+    /** This command enables a certain action or automation with a duration. */
+    enableActionWithDuration: BitFlag(9),
+
+    /** This command disables a certain action or automation. */
+    disableAction: BitFlag(10),
+
+    /** This command disables a certain action or automation with a duration. */
+    disableActionWithDuration: BitFlag(11),
+
+});
+
+/**
+ * Action error.
+ *
+ * @see {@link MatterCoreSpecificationV1_0} § 9.14.9.3
+ */
+export const enum ActionErrorEnum {
+    Unknown = 0,
+
+    Interrupted = 1,
+
+}
+
+/**
+ * Action state.
+ *
+ * @see {@link MatterCoreSpecificationV1_0} § 9.14.9.3
+ */
+export const enum ActionStateEnum {
+    /** The action is not active */
+    Inactive = 0,
+
+    /** The action is active */
+    Active = 1,
+
+    /** The action has been paused */
+    Paused = 2,
+
+    /** The action has been disabled */
+    Disabled = 3,
+}
+
+/**
+ * Types of actions.
+ *
+ * @see {@link MatterCoreSpecificationV1_0} § 9.14.9.2
+ */
+export const enum ActionTypeEnum {
+    /** Use this only when none of the other values applies */
+    Other = 0,
+
+    /** Bring the endpoints into a certain state */
+    Scene = 1,
+
+    /** A sequence of states with a certain time pattern */
+    Sequence = 2,
+
+    /** Control an automation (e.g. motion sensor controlling lights) */
+    Automation = 3,
+
+    /** Sequence that will run when something doesn’t happen*/
+    Exception = 4,
+
+    /** Use the endpoints to send a message to user*/
+    Notification = 5,
+
+    /** Higher priority notification */
+    Alarm = 6,
+}
+
+/**
+ * Provides information about endpoint conformance to a release of a device type definition.
+ *
+ * @see {@link MatterCoreSpecificationV1_0} § 9.14.6
+ */
+const TlvAction = TlvObject({
```

Comment:
Adding the missing comments would be cool

Reply:
Done.